store/sqlstore/supplier_reactions.go

```diff
@@ -189,12 +189,12 @@ func deleteReactionAndUpdatePost(transaction *gorp.Transaction, reaction *model.
 		return err
 	}
 
-	return updatePostForReactions(transaction, reaction.PostId)
+	return updatePostForReactionsOnDelete(transaction, reaction.PostId)
 }
 
 const (
 	// Set HasReactions = true if and only if the post has reactions, update UpdateAt only if HasReactions changes
-	UPDATE_POST_HAS_REACTIONS_QUERY = `UPDATE
+	UPDATE_POST_HAS_REACTIONS_ON_DELETE_QUERY = `UPDATE
```

Comment:
Can we be consistent about where these SQL statements are? Usually we just inline them.

Reply:
Yes, I used the inlined version in my own query, but the original query was declared in a constant to use it in other place before. I didn't modify anything there

Reply:
Here, the explanation about having one constant, instead of an inline SQL, is that SQL is used in 2 places, and was there before I modify the code, I only renamed the constant.